pkg/bundle/resources.go

```diff
@@ -360,6 +362,10 @@ func readContent(ctx context.Context, progress *progress.Progress, base, name st
 	}
 
 	err = filepath.Walk(temp, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
@prachidamle can you comment on this, too? 

I was under the impression that this is a case, where we want to stop, but I can't be sure: 
https://github.com/manno/fleet/pull/2#discussion_r870022658

Reply:
yes, we can error out here and that will catch any problems if the dir is inaccessible. We are returning error already from this method when the file cannot be read, so adding this should be fine.